pages/index.vue

```diff
@@ -1344,13 +1347,18 @@ main
         letter-spacing -0.005em
         color #989BB9
 
+.nav-container
+  height 63px
+
 .section-nav
-  position sticky
-  position -webkit-sticky
-  top -1px
+  top 0
   width 100%
-  z-index 10
-  backdrop-filter blur(30px)
+  // since most of z-indexes in interoperable-planets are 1000
+  z-index 10000
+  &.headroom--frozen
+    position revert
+  &:not(.headroom--frozen)
```

Comment:
Remember this also applies before headroom has even added the frozen class. So it would go `page loads` --> `user scrolls down past nav` --> `headroom pins` --> `user scrolls up` --> `headroom unpins` --> `user scrolls up past nav origin` --> `headroom freezes`.

We want this background to only apply when it is in the fixed position. I'm pushing a fix, but I really think we should look for a better solution here. I'm not sure headroom is cutting it. This pattern is implemented on countless websites for years, so no need to reinvent the wheel. Let's just look for a solution that works. 😄